chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
+Therefore this is a broadly unsuitable way to prepare technical documents.
+
+There are a number of tools that can be used for dynamic documents.
+They fall into two broad groups --
+the first compiles a document as part of code execution,
+and the second operates a separate document compiler.
+In the first group are tools such as R's RMarkdown and Stata's \texttt{dyndoc}.
+These tools ``knit'' or ``weave'' text and code together,
+and are programmed to insert code outputs in pre-specified locations.
+Documents called ``notebooks'' (such as Jupyter) work similarly,
+as they also use the underlying analytical software to create the document.
+These types of dynamic documents are usually appropriate for short or informal materials
+because they tend to offer limited editability outside the base software
+and often have limited abilities to incorporate precision formatting.
+
+On the other hand, some dynamic document tools do not require
+operation of any underlying software, but simply require
+that the writer have access to the updated outputs.
+One very simple one is Dropbox Paper, a free online writing tool
+that allows linkages to files in Dropbox,
+which are then automatically updated anytime the file is replaced.
+Like the first class of tools, Dropbox Paper has very limited formatting options,
+but it is appropriate for working with collaborators who are not using statistical software.
+However, the most widely utilized software
+for dynamically managing both text and results is \LaTeX\ (pronounced ``lah-tek'').\sidenote{
+  \url{https://www.maths.tcd.ie/~dwilkins/LaTeXPrimer/GSWLaTeX.pdf}}
+  \index{\LaTeX}
+While this tool has a significant learning curve,
+its enormous flexibility in terms of operation, collaboration,
+and output formatting and styling
+makes it the primary choice for most large technical outputs today.
+In fact, \LaTeX\ operates behind-the-scenes in many of the tools listed in the first group.
+
+\subsection{Technical writing with \LaTeX}
+
+\LaTeX\ is billed as a ``document preparation system''.
+What this means is worth unpacking.
+In {\LaTeX}, instead of writing in a ``what-you-see-is-what-you-get'' mode
+as you do in Word or the equivalent,
+you write plain text interlaced with coded instructions for formatting
+(similar in concept to HTML).
+The \LaTeX\ system includes commands for simple markup
+like font styles, paragraph formatting, section headers and the like.
+But it also includes special controls for including tables and figures,
+footnotes and endnotes, complex mathematical notation, and automated bibliography preparation.
+It also allows publishers to apply global styles and templates
+to already-written material, allowing them to reformat entire documents in house styles
+with only a few keystrokes.
+In sum, \LaTeX\ enables automatically-organized documents,
+manages tables and figures dynamically,
+and (because it is written in plain text) can be version-controlled using Git.
```

Comment:
```suggestion
and (because it is written in a plain text format, \texttt{.tex}) can be version-controlled using Git.
```
To distinguish what programmers mean as plain text and what that expression sounds like in regular English (like write something explicitly)

Reply:
39da66f7401d2214b0d83bb66ca8260ce14a28fd